docs/9.0/api/contextMenu.md

```diff
@@ -60,8 +60,8 @@ place in the grid among [other features](@/guides/accessories-and-menus/context-
 Possible values:
 * `true` (to enable default options),
 * `false` (to disable completely)
-* an array of [predefined options](@/guides/accessories-and-menus/context-menu.md#context-menu-with-specific-options),
-* an object [with defined structure](@/guides/accessories-and-menus/context-menu.md#context-menu-with-fully-custom-configuration).
+* an array of [predefined options](@/guides/accessories-and-menus/context-menu.md#contextmenuwithspecificoptions),
```

Comment:
Should be `#context-menu-with-specific-options`.

Reply:
#8440